.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
+          registry-url: 'https://registry.npmjs.org'
+
+      - run: yarn install
+      - run: yarn build
+
+      - name: auto-dist-tag
+        run: |
+          yarn global add auto-dist-tag@1
+          auto-dist-tag --write
```

Comment:
I'm not sure what happened but it looks like this step didn't work: https://github.com/simplabs/qunit-dom/commit/585396cacf0f60a5ae9c9e76d6970b01574056e7/checks?check_suite_id=498999285